### PR TITLE
Make the base activity permission aware

### PR DESCRIPTION
**Pull request by deepueg**

### CODE REVIEW

File: android/lib/src/main/java/com/ern/api/impl/navigation/ElectrodeBaseActivity.java [hunk 1]
```diff
@@ -11,17 +12,20 @@
 import androidx.annotation.LayoutRes;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
+import androidx.annotation.RequiresApi;
```
Comment (friederbluemle) [inline android/lib/src/main/java/com/ern/api/impl/navigation/ElectrodeBaseActivity.java:15]: Looks like this import is unused

File: android/lib/src/main/java/com/ern/api/impl/navigation/ElectrodeBaseActivity.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package com.ern.api.impl.navigation;
 
 import android.content.Intent;
+import android.os.Build;
```
Comment (friederbluemle) [inline android/lib/src/main/java/com/ern/api/impl/navigation/ElectrodeBaseActivity.java:4]: Unused import here too

**Comment by deepueg** [comment]
@friederbluemle Thanks for the review. Will fix this.

**Comment by friederbluemle** [comment]
> @friederbluemle Thanks for the review. Will fix this.

Not urgent, just to keep in mind for a future cleanup ;)

**Pull request merged**